apps/checker/ping.go

```diff
@@ -57,6 +96,7 @@ func Ping(ctx context.Context, client *http.Client, inputData request.CheckerReq
 				Timestamp:   time.Now().UTC().UnixMilli(),
 				URL:         inputData.URL,
 				Message:     fmt.Sprintf("Timeout after %d ms", latency),
+				Timing:      timing,
```

Comment:
Instead of saving `DnsStart` and `DnsDone`, lets only save the milliseconds needed, e.g. `dns` (will save some space on tinybird that's just the information we need)